prep-css-flexbox/align-items.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Flexbox - align-items</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="starwars.css" />
+    <style>
+      /* your styles here 👇 */
+
+      .card-deck {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+      }
+
+      .card-container {
+        width: 33.33%;
+        display: flex;
+        min-height: 100vh;
+        align-items: center;
+        justify-content: center;
```

Comment:
You should be able to remove the `display`, `align-items`, and `justify-content` properties from the `.card-container` and still achieve the desired effect. Although, the `min-height` should be moved to your `#card-deck`.